ts/src/binance.ts

```diff
@@ -3613,6 +3618,11 @@ export default class binance extends Exchange {
          * @returns {Trade[]} a list of [trade structures]{@link https://github.com/ccxt/ccxt/wiki/Manual#public-trades}
          */
         await this.loadMarkets ();
+        const paginate = this.safeValue (params, 'paginate', false);
+        if (paginate) {
+            params = this.omit (params, 'paginate');
+            return await this.fetchPaginatedCallDynamic ('fetchTrades', symbol, since, limit, params);
```

Comment:
```suggestion
            return await this.fetchPaginatedCallDynamic ('fetchTrades', symbol, since, limit, params, 1000);
```
not?  https://binance-docs.github.io/apidocs/spot/en/#compressed-aggregate-trades-list

Reply:
I would even say, that i think such hardcoded values are hard to retain. it would have been perfect ( again:) ) to have `.Features` where we could input i.e. `maxAmounts` for these methods, so, these individual implementations wouldn't need such hard maintenance (whenever something changes, we have to hunt them where we need to change those numbers), while in centralized `.Features` we would just change that number on one place, and then all dependent implemetnations would automatically continue working without need to be altered manually and change those hardcoded values.

Reply:
> not? https://binance-docs.github.io/apidocs/spot/en/#compressed-aggregate-trades-list

in this case 1000 is the default arg

Reply:
ok, if so, here - https://github.com/ccxt/ccxt/blob/e60ffec97392df0dd3d070c1bbde2baf8bb7a649/ts/src/binance.ts#L3298 needs to be unset.

however, tbh, I think including that argument is better. if something changes, and lets say exchange supports 1500 instead of 1000, no-one will notice that nuance and it will be left to default 1000.
i am not a fun of leaving defaults unset..

Reply:
`.Features` would have fixed those problems fully.

Reply:
I think that's an orthogonal subject, so out of context right now